boards/arm/nrf5340_audio_dk_nrf5340/nrf5340_audio_dk_nrf5340_cpuapp_common.dts

```diff
@@ -137,8 +137,68 @@
 
 &pwm0 {
```

Comment:
Was not completely sure about the correct file for these configurations, would not mind a pointer